### ISSUE TITLE
Support for siren state

### ISSUE THREAD

**Issue by sjonez** [tsightler/ring-mqtt#66]
I noticed in the raw JSON output from the security panel device of my ring alarm there is an attribute named "siren" with a normal value of "off" - I have yet to test if this changes to "on" when the alarm is sounding but if so it would be really great if you could add this to ring-mqtt so we can create custom rules based on the alarm siren sounding (e.g. to set off external ring camera sirens, or send custom alerts)

**Comment by tsightler** [comment]
I'm going to close this because I think this is basically the same as #27.   My current thought is to add this support as part of the base station vs the alarm-control-panel.  Provide a switch for the siren (same as test mode but also show siren status), but also to provide a generic senor that shows the alarm state (fire, burglar, co, panic, etc) so that specific automation can be triggered based on the alarm state and not just the state of the siren.  If you trigger just on the siren state than even something like testing to siren will trigger the other actions which seems less than ideal to me.

So, basically your request is on my todo list, but I'm not sure when I'll get to it because I really need to test it and, with current conditions, there's pretty much always half-a-dozen people in the house which makes that really difficult.  Feel free to comment on the other feature if you have specific request that you feel the above approach will not address.
